java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/SSL/SSLEngine.java

```diff
@@ -131,17 +131,13 @@ public void initialize() {
                 //
                 KeyManager[] keyManagers = null;
                 KeyStore keys = null;
-                if (_keystoreStream != null || !keystorePath.isEmpty()) {
```

Comment:
With 3.7 we have setters in the Plugin API for keystore and truststore, but that was remove and this code was not properly cleanup.